Rakefile

```diff
@@ -6,6 +6,45 @@ task :test do
   ensure
     sh "docker-compose kill"
   end
+  sh "python -m tests.beup -nchmark"
+end
+
+desc 'CI dependent task; tasks in parallel'
+task:test_parallel do
+  begin
+    ignore_cassandra = sh "git diff-tree --no-commit-id --name-only -r HEAD | grep ddtrace/contrib/cassandra"
+  rescue StandardError => e
+    ignore_cassandra = false
+  end
+  sh "docker-compose up -d | cat"
+  # If cassandra hasn't been changed ignore cassandra tests
+  if not ignore_cassandra
+    n_total_envs = `tox -l | grep -v cassandra | wc -l`.to_i
+    envs = 'tox -l | grep -v cassandra | tr \'\n\' \',\''
+  else
+    n_total_envs = `tox -l | wc -l`.to_i
+    envs = 'tox -l | tr \'\n\' \',\''
+  end
+  circle_node_tot = ENV['CIRCLE_NODE_TOTAL'].to_i
+  n_envs_chunk = n_total_envs / circle_node_tot
+  env_limiter_one = 1
```

Comment:
instead of env_limiter_one and env_limiter_two , let's call it env_list_start and env_list_end